### PR TITLE
added unique to emails. A user to an email

### PR DESCRIPTION
**Pull request by elseagle**
Emails should be unique for each user

### CODE REVIEW

File: django/contrib/auth/models.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ class AbstractUser(AbstractBaseUser, PermissionsMixin):
         },
     )
     first_name = models.CharField(_('first name'), max_length=30, blank=True)
-    last_name = models.CharField(_('last name'), max_length=150, blank=True)
+    last_name = models.CharField(_('last name'), max_length=150, blank=True, unique =True,)
     email = models.EmailField(_('email address'), blank=True)
```
Comment (auvipy) [inline django/contrib/auth/models.py]: you might want to as unique=True to email field?

Comment (elseagle) [reply django/contrib/auth/models.py]: That's my pull request. Emails are meant to be unique, is there a case scenario where they won't be?

Comment (auvipy) [reply django/contrib/auth/models.py]: look, you added unique true to last name not in email

Comment (elseagle) [reply django/contrib/auth/models.py]: oh, my bad. I've made the correction already

**Comment by timgraham** [comment]
We can't change this due to backwards compatibility. Use a custom user model if your project has this requirement.

**Pull request closed without merging**